common/core/nesteddisplay.cpp

```diff
@@ -393,7 +393,7 @@ bool NestedDisplay::GetDisplayAttribute(uint32_t /*config*/,
       break;
     case HWCDisplayAttribute::kRefreshRate:
       // in nanoseconds
-      *value = 60;
+      *value = 16666666;
       break;
```

Comment:
@txu4  Any comments ?

Reply:
This patch is right, AOSP framework will convert the nanosecond to real refresh rate.